src/PSRule/Pipeline/PipelineBuilder.cs

```diff
@@ -426,5 +426,10 @@ protected void AddVisitTargetObjectAction(VisitTargetObjectAction action)
             var previous = VisitTargetObject;
             VisitTargetObject = (targetObject) => action(targetObject, previous);
         }
+
+        protected static int LimitJsonIndentRange(int? jsonIndent)
```

Comment:
Added this to normalize indentation if it falls outside the 0-4 bounds. Was thinking if throwing `ArgumentOutOfRange` exception, but not sure if thats necessary.